backend/graminsta/post/models.py

```diff
@@ -0,0 +1,29 @@
+"""
+Here defines the database tables for post app
+"""
+
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Post(models.Model):
+    """
+    Post Table
```

Comment:
The documentation should describe what a Post is, instead of saying it's a table.

Like: `A Post is usually user generated which contains information like an image, and some description to the image`